### DISCUSSION TITLE
How to delete particular contribution

### DISCUSSION THREAD

**Discussion by RealRichi3** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#53] [Outreachy 2022] [answered]
![image](https://user-images.githubusercontent.com/76791916/199610895-fe9e8fbf-1b2f-499a-9233-1344e81bcdcc.png)

@BatoolMM  I mistakenly added an already existing contribution twice, is there a way to delete it. I tried but i couldn't

**Comment by BatoolMM** [comment, marked as answer]
Hi @RealRichi3, you will have to email outreachy, if you want to delete it but you shouldn't worry too much about it.

**Comment by RealRichi3** [comment]
Alright, thanks
